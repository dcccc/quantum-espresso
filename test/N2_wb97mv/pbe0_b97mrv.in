 &control
    calculation = 'scf',pseudo_dir="./", disk_io="none"
 /
 &system    
    ibrav=  1, celldm(1) =16, nat=  2, ntyp= 1,
    ecutwfc =30.0,  nbnd = 5,
    input_dft='xc-000i-000i-000i-000i-769l-000i-vv10',
 nqx1=1, nqx2=1, nqx3=1, 
    exxdiv_treatment='none', ace=.false.,
    x_gamma_extrapolation = .false.
 /
 &electrons
 /
ATOMIC_SPECIES
 N  14.0  N.pbe-hgh.UPF
ATOMIC_POSITIONS (crystal)
N      0.0               0.0   0.0 
N      0.14763485355     0.0   0.0 
K_POINTS gamma
