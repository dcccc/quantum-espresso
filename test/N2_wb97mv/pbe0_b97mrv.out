
     Program PWSCF v.7.4 starts on 11Jan2025 at 23:44: 0 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on       1 processor cores
     Number of MPI processes:                 1
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     230475 MiB available memory on the printing compute node when the environment starts
 
     Waiting for input...
     Reading input from standard input

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4
     rvv10 b value from libxc   6.0000000
     rvv10_c value from libxc   0.0093000

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= XC-000I-000I-000I-000I-769L-000I-VV10
                           (   0   0   0   0  26   0 769)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     gamma-point specific algorithms are used
 
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum        2449    2449    609                91047    91047   11363
 
     Using Slab Decomposition
 


     bravais-lattice index     =            1
     lattice parameter (alat)  =      16.0000  a.u.
     unit-cell volume          =    4096.0000 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =        10.00
     number of Kohn-Sham states=            5
     kinetic-energy cutoff     =      30.0000  Ry
     charge density cutoff     =     120.0000  Ry
     scf convergence threshold =      1.0E-06
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= XC-000I-000I-000I-000I-769L-000I-VV10
                           (   0   0   0   0  26   0 769)

     celldm(1)=  16.000000  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.000000 )  


     PseudoPot. # 1 for N  read from file:
     ./N.pbe-hgh.UPF
     MD5 check sum: 1504e9c56d21d37eacabc67df972de26
     Pseudo is Norm-conserving, Zval =  5.0
     Generated in analytical, separable form
     Using radial grid of 1085 points,  1 beta functions with: 
                l(1) =   0

     atomic species   valence    mass     pseudopotential
     N                 5.00    14.00000     N ( 1.00)

      8 Sym. Ops. (no inversion) found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1        N      tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2        N      tau(   2) = (   0.1476349   0.0000000   0.0000000  )

     number of k points=     1
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000

     Dense  grid:    45524 G-vectors     FFT dimensions: (  60,  60,  60)

     Estimated max dynamical RAM per process >      58.25 MB

     WARNING: libxc functional with ID  769 depends
      on external parameters: check the user_guide of
      QE if you need to modify them or to check their
      default values.

     WARNING: libxc functional with ID  769 is CAM, 
     long range exx is not available yet (if needed).
    WARNING: libxc functional with ID  769 is still 
    in development.

     WARNING: libxc functional with ID  769 depends on
      the laplacian of the density, which is currently set
      to zero.

     Initial potential from superposition of free atoms

     starting charge      10.0000, renormalised to      10.0000

     negative rho (up,down):  1.043E-01 0.000E+00


---------------------------------------------------------------------------------
Carrying out rVV10 run using the following parameters:
Nqs =      20    Nr_points =    1024   r_max =   100.000
b_value =  6.00000    beta =  0.00969
     q_mesh =  0.00010000  0.00030000  0.00058939  0.00100810
               0.00161396  0.00249058  0.00375900  0.00559430
               0.00824984  0.01209221  0.01765183  0.02569619
               0.03733578  0.05417739  0.07854596  0.11380545
               0.16482331  0.23864234  0.34545298  0.50000000

Gradients computed in Reciprocal space

---------------------------------------------------------------------------------


     Starting wfcs are    8 randomized atomic wfcs

     total cpu time spent up to now is        1.6 secs

     per-process dynamical memory:    15.6 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     c_bands:  5 eigenvalues not converged
     ethr =  1.00E-02,  avg # of iterations = 20.0

     negative rho (up,down):  1.126E-03 0.000E+00

     total cpu time spent up to now is        3.1 secs

     total energy              =     208.91631000 Ry
     estimated scf accuracy    <     389.34198680 Ry

     iteration #  2     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations = 14.0

     negative rho (up,down):  6.380E-04 0.000E+00

     total cpu time spent up to now is        4.3 secs

     total energy              =     -45.11935767 Ry
     estimated scf accuracy    <     210.02760671 Ry

     iteration #  3     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  3.0

     negative rho (up,down):  3.696E-04 0.000E+00

     total cpu time spent up to now is        5.0 secs

     total energy              =     -40.19075294 Ry
     estimated scf accuracy    <      69.93989587 Ry

     iteration #  4     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.0

     negative rho (up,down):  1.680E-04 0.000E+00

     total cpu time spent up to now is        5.8 secs

     total energy              =     -38.54307499 Ry
     estimated scf accuracy    <      25.14834128 Ry

     iteration #  5     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.0

     negative rho (up,down):  7.146E-06 0.000E+00

     total cpu time spent up to now is        6.5 secs

     total energy              =     -38.04315529 Ry
     estimated scf accuracy    <       6.82953056 Ry

     iteration #  6     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.0

     negative rho (up,down):  6.895E-06 0.000E+00

     total cpu time spent up to now is        7.3 secs

     total energy              =     -38.15927637 Ry
     estimated scf accuracy    <       0.09607567 Ry

     iteration #  7     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  9.61E-04,  avg # of iterations =  1.0

     negative rho (up,down):  1.036E-03 0.000E+00

     total cpu time spent up to now is        8.0 secs

     total energy              =     -38.15995800 Ry
     estimated scf accuracy    <       0.13927995 Ry

     iteration #  8     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  9.61E-04,  avg # of iterations =  2.0

     negative rho (up,down):  4.823E-03 0.000E+00

     total cpu time spent up to now is        8.7 secs

     total energy              =     -38.16682595 Ry
     estimated scf accuracy    <       0.00466134 Ry

     iteration #  9     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     c_bands:  4 eigenvalues not converged
     ethr =  4.66E-05,  avg # of iterations = 20.0

     negative rho (up,down):  4.873E-03 0.000E+00

     total cpu time spent up to now is       10.0 secs

     total energy              =      65.34835549 Ry
     estimated scf accuracy    <     562.01929181 Ry

     iteration # 10     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     c_bands:  1 eigenvalues not converged
     ethr =  4.66E-05,  avg # of iterations = 20.0

     negative rho (up,down):  2.482E-04 0.000E+00

     total cpu time spent up to now is       11.2 secs

     total energy              =     162.56133324 Ry
     estimated scf accuracy    <     968.05656265 Ry

     iteration # 11     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.66E-05,  avg # of iterations = 20.0

     negative rho (up,down):  6.843E-05 0.000E+00

     total cpu time spent up to now is       12.5 secs

     total energy              =     -38.16739001 Ry
     estimated scf accuracy    <       0.00179631 Ry

     iteration # 12     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.80E-05,  avg # of iterations =  1.0

     negative rho (up,down):  1.359E-03 0.000E+00

     total cpu time spent up to now is       13.2 secs

     total energy              =     -38.16675862 Ry
     estimated scf accuracy    <       0.00249482 Ry

     iteration # 13     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.80E-05,  avg # of iterations =  3.0

     negative rho (up,down):  1.300E-03 0.000E+00

     total cpu time spent up to now is       14.0 secs

     total energy              =     -38.16722687 Ry
     estimated scf accuracy    <       0.00189904 Ry

     iteration # 14     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.80E-05,  avg # of iterations =  1.0

     negative rho (up,down):  9.111E-04 0.000E+00

     total cpu time spent up to now is       14.8 secs

     total energy              =     -38.16693835 Ry
     estimated scf accuracy    <       0.00164498 Ry

     iteration # 15     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     c_bands:  2 eigenvalues not converged
     ethr =  1.64E-05,  avg # of iterations = 20.0

     negative rho (up,down):  9.300E-04 0.000E+00

     total cpu time spent up to now is       16.0 secs

     total energy              =      62.01980769 Ry
     estimated scf accuracy    <     537.44179916 Ry

     iteration # 16     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     c_bands:  3 eigenvalues not converged
     ethr =  1.64E-05,  avg # of iterations = 20.0

     negative rho (up,down):  9.579E-04 0.000E+00

     total cpu time spent up to now is       17.2 secs

     total energy              =     -38.12045778 Ry
     estimated scf accuracy    <       0.19744244 Ry

     iteration # 17     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.64E-05,  avg # of iterations =  8.0

     negative rho (up,down):  8.167E-04 0.000E+00

     total cpu time spent up to now is       18.1 secs

     total energy              =     -38.16713675 Ry
     estimated scf accuracy    <       0.00010278 Ry

     iteration # 18     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations =  5.0

     negative rho (up,down):  7.914E-04 0.000E+00

     total cpu time spent up to now is       18.9 secs

     total energy              =     -38.16706146 Ry
     estimated scf accuracy    <       0.00123330 Ry

     iteration # 19     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations =  5.0

     negative rho (up,down):  2.486E-04 0.000E+00

     total cpu time spent up to now is       19.7 secs

     total energy              =     -38.16713863 Ry
     estimated scf accuracy    <       0.00035856 Ry

     iteration # 20     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations =  1.0

     negative rho (up,down):  5.532E-05 0.000E+00

     total cpu time spent up to now is       20.4 secs

     total energy              =     -38.16717798 Ry
     estimated scf accuracy    <       0.00046598 Ry

     iteration # 21     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations =  2.0

     negative rho (up,down):  3.769E-05 0.000E+00

     total cpu time spent up to now is       21.1 secs

     total energy              =     -38.16713162 Ry
     estimated scf accuracy    <       0.00045355 Ry

     iteration # 22     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations =  2.0

     negative rho (up,down):  3.607E-05 0.000E+00

     total cpu time spent up to now is       21.9 secs

     total energy              =     -38.16744969 Ry
     estimated scf accuracy    <       0.00042159 Ry

     iteration # 23     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations =  7.0

     negative rho (up,down):  3.571E-05 0.000E+00

     total cpu time spent up to now is       22.7 secs

     total energy              =     -10.06633539 Ry
     estimated scf accuracy    <      59.13476188 Ry

     iteration # 24     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations = 16.0

     negative rho (up,down):  1.698E-05 0.000E+00

     total cpu time spent up to now is       23.7 secs

     total energy              =     -38.16713580 Ry
     estimated scf accuracy    <       0.00034579 Ry

     iteration # 25     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations =  1.0

     negative rho (up,down):  7.702E-05 0.000E+00

     total cpu time spent up to now is       24.4 secs

     total energy              =     -38.16712340 Ry
     estimated scf accuracy    <       0.00034583 Ry

     iteration # 26     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations =  3.0

     negative rho (up,down):  1.237E-05 0.000E+00

     total cpu time spent up to now is       25.2 secs

     total energy              =     -38.16643987 Ry
     estimated scf accuracy    <       0.00012962 Ry

     iteration # 27     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-06,  avg # of iterations =  2.0

     negative rho (up,down):  1.392E-05 0.000E+00

     total cpu time spent up to now is       25.9 secs

     total energy              =     -38.16712055 Ry
     estimated scf accuracy    <       0.00006154 Ry

     iteration # 28     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  6.15E-07,  avg # of iterations =  1.0

     negative rho (up,down):  1.211E-05 0.000E+00

     total cpu time spent up to now is       26.7 secs

     total energy              =     -38.16712225 Ry
     estimated scf accuracy    <       0.00005090 Ry

     iteration # 29     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  5.09E-07,  avg # of iterations =  1.0

     negative rho (up,down):  1.140E-05 0.000E+00

     total cpu time spent up to now is       27.4 secs

     total energy              =     -38.16712615 Ry
     estimated scf accuracy    <       0.00002647 Ry

     iteration # 30     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.65E-07,  avg # of iterations =  2.0

     negative rho (up,down):  1.047E-05 0.000E+00

     total cpu time spent up to now is       28.1 secs

     total energy              =     -38.16711913 Ry
     estimated scf accuracy    <       0.00001618 Ry

     iteration # 31     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.62E-07,  avg # of iterations =  2.0

     total cpu time spent up to now is       28.9 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (  5682 PWs)   bands (ev):

   -27.9665 -15.5219 -10.1259 -10.1259 -10.1098

     highest occupied level (ev):   -10.1098

!    total energy              =     -38.16713330 Ry
     estimated scf accuracy    <       0.00000038 Ry

     The total energy is the sum of the following terms:
     one-electron contribution =     -66.95017544 Ry
     hartree contribution      =      34.35432463 Ry
     xc contribution           =      -9.15255176 Ry
     ewald contribution        =       3.58126928 Ry

     convergence has been achieved in  31 iterations
     Message from routine punch:
     Nothing written, as per user request (disk_io='none')
 
     init_run     :      1.48s CPU      1.61s WALL (       1 calls)
     electrons    :     25.26s CPU     27.21s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.05s CPU      0.06s WALL (       1 calls)
     potinit      :      1.38s CPU      1.48s WALL (       1 calls)
     hinit0       :      0.03s CPU      0.06s WALL (       1 calls)

     Called by electrons:
     c_bands      :      7.29s CPU      7.31s WALL (      31 calls)
     sum_band     :      0.85s CPU      0.86s WALL (      31 calls)
     v_of_rho     :     17.70s CPU     19.69s WALL (      32 calls)
     mix_rho      :      0.68s CPU      0.71s WALL (      31 calls)

     Called by c_bands:
     init_us_2    :      0.02s CPU      0.02s WALL (      63 calls)
     regterg      :      7.28s CPU      7.29s WALL (      31 calls)

     Called by *egterg:
     rdiaghg      :      0.02s CPU      0.02s WALL (     240 calls)
     h_psi        :      7.12s CPU      7.15s WALL (     241 calls)
     g_psi        :      0.01s CPU      0.01s WALL (     209 calls)

     Called by h_psi:
     h_psi:calbec :      0.04s CPU      0.04s WALL (     241 calls)
     vloc_psi     :      1.87s CPU      1.88s WALL (     241 calls)
                                        0.00s GPU  (     241 calls)
     add_vuspsi   :      0.01s CPU      0.01s WALL (     241 calls)
     h_psi_meta   :      5.20s CPU      5.21s WALL (     241 calls)

     General routines
     calbec       :      0.04s CPU      0.04s WALL (     241 calls)
     fft          :      2.56s CPU      2.61s WALL (    1917 calls)
     ffts         :      0.08s CPU      0.08s WALL (      62 calls)
     fftw         :      6.15s CPU      6.17s WALL (    4836 calls)
     interpolate  :      0.01s CPU      0.01s WALL (      32 calls)
 
     Parallel routines
 
     PWSCF        :     26.77s CPU     28.86s WALL

 
   This run was terminated on:  23:44:29  11Jan2025            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
